aarch64_linux/aarch64_wheel_ci_build.py

```diff
@@ -9,103 +9,201 @@
     parser.add_argument("--build-only", action="store_true")
     parser.add_argument("--test-only", type=str)
     parser.add_argument("--enable-mkldnn", action="store_true")
+    parser.add_argument("--enable-cuda", action="store_true")
     return parser.parse_args()
 
 
-if __name__ == '__main__':
-    '''
+if __name__ == "__main__":
+    """
     Entry Point
-    '''
+    """
     args = parse_arguments()
     enable_mkldnn = args.enable_mkldnn
-    repo = Repository('/pytorch')
+    enable_cuda = args.enable_cuda
+    repo = Repository("/pytorch")
     branch = repo.head.name
-    if branch == 'HEAD':
-        branch = 'master'
-
+    if branch == "HEAD":
+        branch = "master"
 
-    print('Building PyTorch wheel')
+    print("Building PyTorch wheel")
     build_vars = "CMAKE_SHARED_LINKER_FLAGS=-Wl,-z,max-page-size=0x10000 "
     os.system("python setup.py clean")
 
     override_package_version = os.getenv("OVERRIDE_PACKAGE_VERSION")
     if override_package_version is not None:
         version = override_package_version
-        build_vars += f"BUILD_TEST=0 PYTORCH_BUILD_VERSION={version} PYTORCH_BUILD_NUMBER=1 "
-    elif branch in ['nightly', 'master']:
-        build_date = check_output(['git', 'log', '--pretty=format:%cs', '-1'], cwd='/pytorch').decode().replace('-', '')
-        version = check_output(['cat', 'version.txt'], cwd='/pytorch').decode().strip()[:-2]
+        build_vars += (
+            f"BUILD_TEST=0 PYTORCH_BUILD_VERSION={version} PYTORCH_BUILD_NUMBER=1 "
+        )
+    elif branch in ["nightly", "master"]:
+        build_date = (
+            check_output(["git", "log", "--pretty=format:%cs", "-1"], cwd="/pytorch")
+            .decode()
+            .replace("-", "")
+        )
+        version = (
+            check_output(["cat", "version.txt"], cwd="/pytorch").decode().strip()[:-2]
+        )
         build_vars += f"BUILD_TEST=0 PYTORCH_BUILD_VERSION={version}.dev{build_date} PYTORCH_BUILD_NUMBER=1 "
     elif branch.startswith(("v1.", "v2.")):
         build_vars += f"BUILD_TEST=0 PYTORCH_BUILD_VERSION={branch[1:branch.find('-')]} PYTORCH_BUILD_NUMBER=1 "
 
     if enable_mkldnn:
         build_ArmComputeLibrary()
         print("build pytorch with mkldnn+acl backend")
-        build_vars += "USE_MKLDNN=ON USE_MKLDNN_ACL=ON " \
-                      "ACL_ROOT_DIR=/acl " \
-                      "LD_LIBRARY_PATH=/pytorch/build/lib:/acl/build:$LD_LIBRARY_PATH " \
-                      "ACL_INCLUDE_DIR=/acl/build " \
-                      "ACL_LIBRARY=/acl/build "
+        build_vars += (
+            "USE_MKLDNN=ON USE_MKLDNN_ACL=ON "
+            "ACL_ROOT_DIR=/acl "
+            "LD_LIBRARY_PATH=/pytorch/build/lib:/acl/build:$LD_LIBRARY_PATH "
+            "ACL_INCLUDE_DIR=/acl/build "
+            "ACL_LIBRARY=/acl/build "
+        )
     else:
         print("build pytorch without mkldnn backend")
 
     os.system(f"cd /pytorch; {build_vars} python3 setup.py bdist_wheel")
```

Comment:
I see you are packaging the cuda libs post torch build, don't we need to build pytorch with `USE_CUDA=1` if cuda is enabled?